freedomofpress/securedrop
# Update Tails desktop icons, updater banner, updater icon

These were still using the old logo. Note that `resources_rc.py` needed to be regenerated for these updates to be reflected in the updater UI.

Resolves #4788

## Status

Ready for review 

## Test plan

- Run the updater in a Tails environment
- Run `securedrop-admin tailsconfig` in a Tails environment
- Ensure that updated graphics for desktop shortcuts, graphical updater icon, and graphical updater banner are used, with no visual defects
